### PR TITLE
Add DNA check to the Baby and notification

### PR DESCRIPTION
**Pull request by JeffGainsNGames**

### CODE REVIEW

File: UndertaleModTool
Comment (Miepee) [inline UndertaleModTool:1]: Please redo this change

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 This format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [1.1.5] - yyyy-mm-dd
```
Comment (Miepee) [inline CHANGELOG.md:6]: if you want to, you can fill in here `2024-01-??`. if not, leave it be

File: YAMS-LIB/Program.cs [hunk 2]
```diff
@@ -3033,6 +3033,15 @@ void RotateTextureAndSaveToTexturePage(int rotation, UndertaleTexturePageItem te
         // Removes cutscenes for type 1's
         gmData.Code.ByName("gml_Object_oItem_Other_10").ReplaceGMLInCode("display_itemmsg", "if (!global.skipItemFanfare) display_itemmsg");
 
+        // Add DNA check to Baby trigger
+        gmData.Code.ByName("gml_Object_oHatchlingTrigger_Collision_267").PrependGMLInCode(
+            """
+            if (global.dna < 46){
```
Comment (Miepee) [inline YAMS-LIB/Program.cs:3039]: please use [Allman](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) bracket style:
```gml
if (global.dna < 46)
{
```

**Comment by JeffGainsNGames** [comment]
resolves #144

**Review by Miepee — APPROVED** [review]

**Pull request merged**